### DISCUSSION TITLE
[Question][Hw6] Request for information in leaderboard

### DISCUSSION THREAD

**Discussion by Lee-Janggun** [prosyslab-classroom/cs424-program-reasoning#301] [Q&A]
Currently, the leaderboard only says Timeout, regardless on whether the actual reason was a timeout or other failures. Is it possible that the leaderboard can be updated so that the two can be distinguished?

**Comment by Hhro** [comment]
Hi, Janggun.
It's a good idea, and also possible.
However, considering the deadline is almost over, it might be unfair to other students who have already completed this homework.
Therefore, I think we should stick to the current grading system and update it for the next synthesis homework.
What do you think, professor @KihongHeo ?

**Comment by Lee-Janggun** [comment]
Thats understandable. Still, thank you for considering the idea for the last homework :)

**Comment by KihongHeo** [comment]
Janggun, That is a good idea.
We will update it for the next homework.

**Comment by Lee-Janggun** [comment]
Thanks for the confirmation!
